src/commands/services/generate.ts

```diff
@@ -0,0 +1,28 @@
+import { flags } from '@oclif/command';
```

Comment:
Looks like there are two new files here that are unrelated to login. Let’s pull those into a separate PR.

Reply:
More than 2. I can pull them out.